benchmarks/benchmarks/bench_lib.py

```diff
@@ -54,3 +54,67 @@ def setup(self, shape, pad_width, mode):
 
     def time_pad(self, shape, pad_width, mode):
         np.pad(self.array, pad_width, mode)
+
+class Nan(Benchmark):
+    """Benchmarks for nan functions"""
+
+    param_names = ["array_size", "percent_nans"]
+    params = [
+            # sizes of the 1D arrays
+            [6, 200, int(2e5)],
```

Comment:
Not sure 200 adds much to 6 - at least on my computer it is less than a 10% increase. Also, with `6`, the percentages are meaningless. Maybe just skip it? That way we have one short and one long, which should cover it.